test/unit/spec/v2/ion/uiSchemaLabelTransformer_spec.js

```diff
@@ -0,0 +1,827 @@
+      'mfa.challenge.password.placeholder': 'password',
+
+      'primaryauth.password.placeholder': 'password',
+      'primaryauth.username.placeholder': 'username',
+      'remember': 'remember me',
+      'security.disliked_food': 'dislike food answer',
+      'security.name_of_first_plush_toy': 'first plush toy answer',
+      'security.favorite_vacation_location': 'vacation location answer',
+
+    }, (value) => `unit test - ${value}`);
+  });
+  afterAll(() => {
+    Bundles.login = originalLoginBundle;
+  });
+
+  it('no remediations', () => {
+    const resp = {
+      stateHandle: 'xxx'
+    };
+    expect(uiSchemaLabelTransformer(resp)).toEqual({
+      stateHandle: 'xxx'
+    });
+  });
+
+  it('no ui schemas', () => {
+    const resp = {
+      remediations: [
+        {
+          name: 'test',
+          value: []
+        }
+      ]
+    };
+    expect(uiSchemaLabelTransformer(resp)).toEqual({
+      remediations: [
+        {
+          name: 'test',
+          value: []
+        }
+      ]
+    });
+  });
+
+  it('converts label for identify with password', () => {
+    const resp = {
+      remediations: [
+        {
+          name: 'identify',
+          uiSchema: [
+            {
+              label: 'Username',
+              'label-top': true,
+              name: 'identifier',
+              type: 'text',
+            },
+            {
+              label: 'Password',
+              'label-top': true,
+              name: 'credentials.passcode',
+              params: { showPasswordToggle: true },
+              secret: true,
+              type: 'password',
+            },
+            {
+              label: false,
+              'label-top': true,
+              modelType: 'boolean',
+              name: 'rememberMe',
+              placeholder: 'Remember Me',
+              required: false,
+              type: 'checkbox',
+            }
+          ]
+        }
+      ]
+    };
+    expect(uiSchemaLabelTransformer(resp)).toEqual({
+      remediations: [
+        {
+          name: 'identify',
+          uiSchema: [
+            {
+              label: 'unit test - username',
+              'label-top': true,
+              name: 'identifier',
+              type: 'text',
+            },
+            {
+              label: 'unit test - password',
+              'label-top': true,
+              name: 'credentials.passcode',
+              params: { showPasswordToggle: true },
+              secret: true,
+              type: 'password',
+            },
+            {
+              label: false,
+              'label-top': true,
+              modelType: 'boolean',
+              name: 'rememberMe',
+              placeholder: 'unit test - remember me',
+              required: false,
+              type: 'checkbox',
+            }
+          ]
+        }
+      ]
+    });
+  });
+
+  it('converts label for select-authenticator-authenticate', () => {
+    const resp = {
+      remediations: [
+        {
```

Comment:
For this the strings in factorUtil will still take the priority right ?

Reply:
yeah, I'm thinking to remove the label override in FactorUtil (in another PR). what do you think?

Reply:
Might be tricky. OV needs some special things. Lets discuss.

Reply:
yeah, once we need to support multi authenticator in select authenticator page, the override doesn't work correctly.